stubs/TYPO3/CMS/Core/Database/Query/Expression/ExpressionBuilder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
```

Comment:
According to code style, there should be an empty line before and after declare statements.